tpgtools/overrides/orgpolicy/beta/policy.yaml

```diff
@@ -0,0 +1,10 @@
+- type: CUSTOM_IMPORT_FUNCTION
```

Comment:
The generated sweeper does not seem to work correctly likely tied to the need for the custom import. We should probably add a `NO_SWEEPER` override for now.

Reply:
Ah, yes, thank you!

Reply:
Added.